### PR TITLE
Use redux to manage state

### PR DESCRIPTION
**Pull request by dscottboggs**
While I'm not super impressed with the ease-of-use of Redux, it seems like a more sensible and well-optimized way to manage state than passing callbacks down through props. It allows, for example, `LoginForm` to be totally props-free, rather than needing a callback passed down from `LoginView` which calls another callback passed down from `Screen`. These advantages will only become more clear as the project expands, and I think it's sensible to adopt this standardized structure from the beginning.

### CODE REVIEW

**Comment by j8r** [comment]
Sorry, I won't be able to review much on this project. I have no knowledge of React/Redux, and my JS one is very limited (which is a strong prerequesite for using React).

I am afraid that using React/Redux will have a steeper learning curve than using other alternatives, which will limit contributions from newbies like me.

I would like to experiment with https://github.com/sveltejs/svelte too before setling definitely with React.

I propose having a webapp with a login form + available applications view (/src endpoint) for both JS framework, then decide it which one we pick.

**Comment by dscottboggs** [comment]
Sure, I was hoping to have the opportunity to try out svelte anyway. However, Redux is a framework-independent library for managing state. The process is straightforward once you can understand the structure, and it may be worth considering regardless of the framework.

**Comment by dscottboggs** [comment]
Looking through the examples, svelte looks very intuitive. :smiley:

File: src/index.html [hunk 1]
```diff
@@ -8,9 +8,7 @@
 </head>
 
 <body>
-  <div id="react-root">
-  </div>
-  <script src="/build/bundle.js" charset="utf-8"></script>
+  <div id="react-root"> </div>
 </body>
 
 </html>
```
Comment (j8r) [inline src/index.html]: newline missing

File: src/components/Screen.jsx [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
\ No newline at end of file
+const Screen = props => (
+    <div className='container'>
+        {props.loggedIn ? <MainView /> : <LoginView />}
+    </div>
+)
+
+Screen.propTypes = { loggedIn: PropTypes.bool.isRequired }
+
+export default connect(detectLogin)(Screen)
```
Comment (j8r) [inline src/components/Screen.jsx]: newline mssing

**Comment by j8r** [comment]
There is also [Sapper](https://sapper.svelte.dev/), built on top of Svelte ([example](https://github.com/sveltejs/sapper-template). They use also the term fixtures (https://github.com/sveltejs/sapper-template/tree/master/cypress/fixture) :smile: 
Do you think we better use a vanilla Svelte, or we can benefit from Sapper?

**Comment by dscottboggs** [comment]
> Some newlines are missing - what text editor are you using?

VSCode... apparently "insert final newlines" is an option that's off by default. I've turned it on now, sorry about that :roll_eyes:

**Pull request merged**